data_validation/cli_tools.py

```diff
@@ -170,6 +170,10 @@ def _configure_find_tables(subparsers):
     find_tables_parser.add_argument(
         "--allowed-schemas", "-as", help="List of source schemas to match."
     )
+    find_tables_parser.add_argument(
```

Comment:
Can we document this argument in the README?

Reply:
done